src/components/staking/Undelegations.vue

```diff
@@ -1,12 +1,5 @@
 <template>
-  <div
-    v-if="
-      !$apollo.queries.undelegations.loading &&
-        undelegations &&
-        undelegations.length > 0
-    "
-    class="undelegations"
-  >
+  <div v-if="undelegations && undelegations.length > 0" class="undelegations">
```

Comment:
this just skips the loading check 🤷‍♂

Reply:
yes, but I think this might be the issue, what got users confused...